src/EchoToImageLUT.hpp

```diff
@@ -33,7 +33,19 @@ namespace radar_base {
 
     public:
         EchoToImageLUT() = delete;
-
+        /**
+         * @brief Returns a pair of iterators pointing to the initial and the final pixel
+         * in a radar echo dot represented by an angle index and a sweep index at the
+         * echoes to image look-up table
+         *
+         * @param angle_idx The angle index
+         * @param sweep_idx The sweep index
+         * @return std::pair<std::vector<cv::Point>::const_iterator,
+         * std::vector<cv::Point>::const_iterator>
+         */
+        std::pair<std::vector<cv::Point>::const_iterator,
+            std::vector<cv::Point>::const_iterator>
+        getPixels(int angle_idx, int sweep_idx);
```

Comment:
When I see a _getter_ method, I expect it to not change its object. I.e. that it is constant
```suggestion
        getPixels(int angle_idx, int sweep_idx) const;
```